langchain4j-core/src/main/java/dev/langchain4j/agent/tool/Tool.java

```diff
@@ -36,4 +36,15 @@
      * @return description of the tool.
      */
     String[] value() default "";
+
+    /**
+     * Indicates whether the tool's String return value should be returned directly to the user.
+     * Only applicable if used within an AiService and if the return type is 'Result<String>'
+     * If true, the result will be returned directly without further LLM processing.
+     * If false or if return type is not Result<String>, normal processing continues.
```

Comment:
Could you please add an explicit validation that checks that if there is at lest a single tool with `directReturn=true`, then return type of such AI Service method should be `Result<String>`? If it is not, we should fail